src/app/application.cpp

```diff
@@ -221,29 +221,22 @@ void Application::setFileLoggerDeleteOld(bool value)
 
 int Application::fileLoggerMaxSize() const
 {
-    int val = settings()->loadValue(KEY_FILELOGGER_MAXSIZE, 10).toInt();
-    if (val < 1)
-        return 1;
-    if (val > 1000)
-        return 1000;
-    return val;
+    int val = settings()->loadValue(KEY_FILELOGGER_MAXSIZEBYTES, 65 * 1024).toInt();
+    return std::min(std::max(val, 1024), 1000 * 1024 * 1024);
```

Comment:
just a note, c++17 have `std::clamp()` for clamping values.
maybe we can add a matching utility function in `Utils` namespace later.

Reply:
It's bad to duplicate these numbers. You need to declare constants: MIN_FILESIZE, MAX_FILESIZE and DEFAULT_FILESIZE.

Reply:
>It's bad to duplicate these numbers. You need to declare constants: MIN_FILESIZE, MAX_FILESIZE and DEFAULT_FILESIZE.

OK. I'll do it later.